completions/bash/oci-runtime-tool

```diff
@@ -119,6 +119,13 @@ __oci-runtime-tool_complete_log_level() {
 	" -- "$cur" ) )
 }
 
+__oci-runtime-tool_complete_compliance_level() {
+	COMPREPLY=( $( compgen -W "
+		must
+		should
```

Comment:
> In this case, we 'may' print 'MAY' information when `platform.os` is `linux` but the `linux` item is not set.

So would this be all unset OPTIONAL child properties?  Or more case-by case at the runtime-tools maintainers descretion?  Either was, a fatal level seems odd.  If something deserves occasional fatal warnings, it should probably be SHOULD in the spec.